.github/workflows/test.yml

```diff
@@ -28,10 +29,14 @@ jobs:
             python-version: 3.8
           - requirements-file: django-5.0.txt
             python-version: 3.9
+          - requirements-file: django-5.1.txt
+            python-version: 3.8
+          - requirements-file: django-5.1.txt
+            python-version: 3.9
 
     services:
       postgres:
-        image: postgres:12
+        image: postgres:13
```

Comment:
@fsbraun Do you think it make sense that we test again an array of postgres version or should be just track the latest or the latest-1 (Currently that would be postgres:17 or postgres:16)